### ISSUE TITLE
Calculate the F-Score for the Logistic Regression and Neural Network

### ISSUE THREAD

**Issue by nrao57** [nrao57/H1B-Visa-Petitions-#1]
Because the H1B Visa target label classes are vastly different in proportion (one class has an order of magnitude more training examples), accuracy is not a good metric for the binary classifier. 

Instead, an F1-Score should be used from the scikit-learn library. The documentation can be found [here](http://scikit-learn.org/stable/modules/generated/sklearn.metrics.f1_score.html)
